articles/fleet-4.50.0.md

```diff
@@ -0,0 +1,137 @@
+
+
+## Mac Admins SOFA support
+
+Fleet has integrated support for the Mac Admins [SOFA](https://github.com/macadmins/sofa) (Structured Open Feed Aggregator), enhancing its capabilities to provide comprehensive tracking and surfacing of update information for macOS and iOS environments. SOFA, known for its machine-readable feed and user-friendly web interface, offers continuous updates on XProtect data, OS updates, and detailed release information. This integration within Fleet is facilitated through the recent updates to the [Mac Admins osquery extension](https://github.com/macadmins/osquery-extension), which now includes tables specifically for security release information (`sofa_security_release_info`) and unpatched CVEs (`sofa_unpatched_cves`).
+
+These additions provide Fleet users with valuable tools for monitoring security updates and vulnerability statuses directly within the Fleet environment. Users can access the new SOFA tables at [SOFA Security Release Info](https://fleetdm.com/tables/sofa_security_release_info) and [SOFA Unpatched CVEs](https://fleetdm.com/tables/sofa_unpatched_cves) for detailed insights. For those looking to delve deeper into the application of these tools, Graham Gilbert’s blog post, [Investigating unpatched CVEs with osquery and SOFA](https://grahamgilbert.com/blog/2024/05/03/investigating-unpatched-cves-with-osquery-and-sofa/), offers an in-depth look at leveraging osquery in conjunction with SOFA to enhance digital security and compliance efforts. This integration underscores Fleet's commitment to providing robust, actionable intelligence for IT administrators and security professionals managing Apple devices.
+
+
+## `zsh` support
+
+Fleet has expanded its scripting capabilities by adding support for `zsh` (Z Shell) scripts, catering to IT administrators' and developers' diverse scripting preferences. This update allows users to execute `zsh` scripts directly within Fleet, providing a flexible and powerful toolset for managing and automating tasks across various systems. By accommodating `zsh`, known for its robust features and interactive use enhancements over `bash`, Fleet enhances its utility for more sophisticated script operations. This support not only broadens the scope of administrative scripts that can be run but also aligns with the ongoing efforts to adapt to the evolving needs of users in dynamic IT environments.
+
+
+
+
+
+## Changes
+
+### Endpoint Operations
```

Comment:
```suggestion
### Endpoint operations
```

[Sentence case](https://fleetdm.com/handbook/company/why-this-way#why-does-fleet-use-sentence-case)